charts/verdaccio/Chart.yaml

```diff
@@ -1,8 +1,8 @@
 apiVersion: v2
-description: A lightweight private npm proxy registry (sinopia fork)
+description: A lightweight private node.js proxy registry
```

Comment:
Is it a proxy though? You can publish to Verdaccio so it is not a proxy, it is just a registry. It is a Node.js registry.

Reply:
You can also use it as a pull-through cache/proxy of upstream registries

Reply:
I know but that is only one of the many features.

Reply:
I understand both points, actually, the web even says `npm`. 🤦🏼 for quite a while, anyway at this point everywhere says `private proxy registry`, blog, readmes even talks 😋 . My change was more oriented on remove `npm` and `sinopia` from the sentence, but maybe in the close future, we can rephrase better the description in a better way.